lib/nrf/common/i2c_common.c

```diff
@@ -180,5 +186,14 @@ void i2c_resume(uint32_t i2c)
 	PERIPH_TRIGGER_TASK(I2C_TASK_RESUME(i2c));
 }
 
-
+/** Configure event -> task shortcuts
+ *
+ * Sets new shortcut configuration for I2C peripheral.
+ * @param[in] i2c i2c peripheral base.
+ * @param[in] shorts shortcut activation configuration
```

Comment:
does this have a @ref it could refer to?

Reply:
Sure, done